invenio_rdm_records/records/mappings/v6/rdmrecords/drafts/draft-v1.0.0.json

```diff
@@ -12,20 +12,47 @@
         },
         "access" : {
           "properties" : {
-            "access_right" : {
+            "record" : {
               "type" : "keyword"
             },
-            "created_by" : {
-              "type" : "long"
+            "files" : {
+              "type" : "keyword"
+            },
+            "owned_by" : {
+              "properties": {
+                "user": {
```

Comment:
Comment: We should check that this structure is fine for the queries we want to make. E.g. we want to query for all records owned by a user - something along the lines of: ``access.owned_by.user:1``. From top of my head, it looks fine, I just want to be sure.

Reply:
quick tests are looking good:

for one demo record, i added `{"user": 2}` (in addition to the existing `{"user": 1}`) to the `access.owned_by`, and in another i replaced `{"user": 1}` by `{"user": 2}`
afterwards, i tried querying for `access.owned_by.user:1` and `...:2` in the search UI (`/search?q=access.owned_by.user%3A2`)

for `user:1`, the expected 99 results were shown, and for `user:2` the expected 2 were shown